Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION = 1.0.0
+VERSION = 1.0.1
```

Comment:
I think we should only be bumping the version when cutting a release from master.